triangles.js

```diff
@@ -0,0 +1,39 @@
+pry = require('pryjs')
+function bubbleSort(arr){
+  var swapped = true;
+  while(swapped) {
+    swapped = false;
+    for(var i = 0; i < arr.length - 1; i++) {
+      if (arr[i] > arr[i + 1]){
+        var bigger = arr[i];
+        arr[i] = arr[i + 1];
+        arr[i + 1] = bigger;
+        swapped = true;
+      };
+    };
+  };
+  return arr;
+};
+
+function isTriangle(side_one, side_two, side_three){
+  var sides = [side_one, side_two, side_three];
```

Comment:
this method is doing a TON of things. could we break this down further and make some single responsibility functions?